Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```
@@ -1878,6 +1878,9 @@
   - type: MeleeChemicalInjector
     transferAmount: 1
     solution: melee # Goobstation - End
+  - type: GibThisGuy  #Goobstation
+    ocNames:
+    - BombasterDS 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)